### ISSUE TITLE
eloquence is compatible with new versions, but not with prior versions

### ISSUE THREAD

**Issue by nunotfc** [pumper42nickel/eloquence_threshold#1]
this new eloquence is fully compatible with NVDA last versions, but not compatible with the prior versions, eg: 2019.2. please can you make it both compatible, as oter addons?

**Comment by lbk2907** [comment]
Hi. Greetings. Sorry. Actually I not have any knoledge in python and
other programming language. What I'm facing now is the NVDA will crash
when eloquence try to read some special character. I will give 2
example below.

1. Ağayev
The character g in the above word actually is not a g. If I use
ESpeak, it will read that letter as g greeve. But if I use the
synthesizer other then ESpeak, it will just read as g.
2. ❤️
The above emoji actually have something that I not sure what is that.
If I move the cursor use the arrow key, NVDA cannot reach it but when
I press backspace, ESpeak will read as symbol f e0f. If I use another
tts, it will just quiet. If I I use the old eloquence, it will read
like when asking question. The following is an example.

The above emoji is called red heart. For example, the sentence is like this.

❤️Hi all! This is an example.

So, the eloquence will read like this.

Red heart? Hi. This is an example.

I cannot provide other example cause I haven't got the character yet.
I'm using NVDA alpha version alpha-18420,a8e8db19.

I hope this can be fixed. Thanks in advance.

Sincerely,
Loh Boon Keat
